.forge-snapshots/Payments_swap_settleWithBalance_takeAllToSpecifiedAddress.snap

```diff
@@ -0,0 +1 @@
+127508
```

Comment:
Having a recipient of a specific address (not a helper constant) is now ~90 more expensive than before. But taking to the `msg.sender` is now ~40 cheaper than before.
And most importantly - our sdks will be much easier to write haha

Reply:
previously taking to any address: 127419
now taking to specific address: 127508
now taking to msg.sender: 127370